gax-java/gax/src/main/java/com/google/api/gax/rpc/ServerStreamingCallSettings.java

```diff
@@ -79,12 +81,14 @@
   @Nonnull private final StreamResumptionStrategy<RequestT, ResponseT> resumptionStrategy;
 
   @Nonnull private final Duration idleTimeout;
+  @Nonnull private final Duration waitTimeout;
```

Comment:
Can we use `java.time.Duration` instead? We no longer support Java 7 and plan to deprecate `org.threeten.bp.Duration` soon.

Reply:
Can this be a separate follow up PR to update the Duration type? I think it'll be cleaner that way.

Reply:
I'm OK with it if the follow up PR is merged before this PR gets released. Otherwise it would be a breaking change to change the type.

Reply:
I see, but idleTimeout is already using `org.threeten.bp.Duration`. So maybe it's worth updating this in the next release? I'm not sure if having mixed Duration types in the same class is a good idea :(

Reply:
Usually it's not, we plan to deprecate `org.threeten.bp.Duration` soon and add an alternative method with `java.time.Duration` for all threeten methods, it would be great if we can change it now, especially if we don't expect customers to set it often.
e.g.  For
```
public Duration getWaitTimeout() {
    return waitTimeout;
  }
```
We will have to add an alternative method
```
public java.time.Duration getWaitTimeoutDuration() {
    return waitTimeout;
  }
```
with a slightly different name as well, which is not ideal.
I'm OK with keeping it this way as well if you feel strongly, but want to share with you our plan and potential consequences.

Reply:
Chatted offline, we'll keep the threeten type for now :)